openapi.yaml

```
@@ -0,0 +1,1791 @@
+    PalletStorageItem:
+      type: object
+      properties:
+        storageItemIndex:
+          $ref: '#/components/schemas/PalletItemMetadataIndex'
+        name:
+          type: string
+        modifier:
+          type: string
+        type:
+          $ref: '#/components/schemas/PalletStorageType'
+        fallback:
+          type: string
+        documentation:
+          type: array
+          items:
+            type: string
+      description: Metadata of a storage item from a FRAME pallet.
+    PalletConstant:
+      type: object
+      properties:
+        constantIndex:
+          $ref: '#/components/schemas/PalletItemMetadataIndex'
+        name:
+          type: string
+        type:
+          type: string
+        value:
+          type: string
+        documentation:
+          type: array
+          items:
+            type: string
+      description: Metadata of a constant from a FRAME pallet.
+    DispatchableArg:
+      type: object
+      properties:
+        name:
+          type: string
+        type:
+          type: string
+      description: Metadata of a parameter to a dispatchable call from a FRAME pallet.
+    Dispatchable:
+      type: object
+      properties:
+        callIndex:
+          $ref: '#/components/schemas/PalletItemMetadataIndex'
+        name:
+          type: string
+        args:
+          type: array
+          items:
+            $ref: '#/components/schemas/DispatchableArg'
+        documentation:
+          type: array
+          items:
+            type: string
+      description: Metadata of a dispatchable call from a FRAME pallet.
+    PalletStorage:
+      type: object
+      properties:
+        prefix:
+          type: string
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/PalletStorageItem'
+    Pallet:
+      type: object
+      properties:
+        index:
+          type: string
+          format: unsignedInteger
+        name:
+          type: string
+        storage:
+          $ref: '#/components/schemas/PalletStorage'
+        calls:
+          type: array
+          items:
+            $ref: '#/components/schemas/Dispatchable'
+        events:
+          type: array
+          items:
+            $ref: '#/components/schemas/PalletEvent'
+        constants:
+          type: array
+          items:
+            $ref: '#/components/schemas/PalletConstant'
+        errors:
+          type: array
+          items:
+            $ref: '#/components/schemas/PalletError'
+      description: Metadata of a FRAME pallet.
+    RuntimeDispatchInfo:
+      type: object
+      properties:
+        weight:
+          type: string
+          description: Extrinsic weight.
+        class:
+          type: string
+          description: Extrinsic class.
+          enum:
+          - Normal
+          - Operational
+          - Mandatory
+        partialFee:
+          type: string
+          description: Transaction fee, not including any tip.
```

Kian and I are trying to drive home the term "inclusion fee".
```suggestion
          description: The _inclusion fee_ of a transaction, i.e. the minimum fee required for a transaction. Includes weight and encoded length fees, but does not have access to any signed extensions, e.g. the `tip`.
```

---

What if we just call it `inclusionFee` instead of partial fee then?